examples/desktop/scatter/scatter_dataslice.py

```diff
@@ -1,40 +1,46 @@
 """
-Scatter Plot
-============
+Scatter Plot Data Slicing
+=========================
+
 Example showing data slice for scatter plot.
 """
 
 # test_example = true
```

Comment:
I thought we want this in docs only but not tests since it creates random points